### PR TITLE
fix: Make the CSS field in DJANGO ADMIN BRANDING a JSON field

### PR DESCRIPTION
**Pull request by Genza999**
This is for the implementation for the CU:
https://app.clickup.com/t/14289126/FC-3272

### CODE REVIEW

**Comment by grizz** [comment]
Task linked: [FC-3272 Make the CSS field in DJANGO ADMIN BRANDING a JSON field](https://app.clickup.com/t/14289126/FC-3272)

File: src/account/views/auth.py [hunk 1]
```diff
@@ -76,18 +75,17 @@ def login(request):
 
     if branding_org:
         org_branding = OrganizationBranding.objects.filter(
-            org__slug=branding_org
+            org_slug=branding_org
```
Comment (vegu) [inline src/account/views/auth.py]: @Genza999 i don't think this change is right

its referencing `slug` through the `org` relationship on the `OrganizationBranding` class. So it should be `__`. Lets actually add a test for this i think.

```
@reversion.register
class OrganizationBranding(HandleRefModel):
    org = models.OneToOneField(Organization, on_delete=models.CASCADE)
```

**Review by vegu — CHANGES_REQUESTED** [review]

File: src/account/views/auth.py (hunk 1 above)
Comment (Genza999) [reply src/account/views/auth.py]: This has been fixed, thanks

Comment (grizz) [reply src/account/views/auth.py]: @Genza999 you need to add tests for that (and any/all changes)

**Comment by vegu** [comment]
@Genza999 this still needs tests added

**Comment by Genza999** [comment]
> @Genza999 this still needs tests added

Tests have been added @vegu

**Review by vegu — APPROVED** [review]

**Pull request merged**